README.md

```diff
@@ -109,6 +109,22 @@ env:
   value: *containerName # referencing earlier anchor in same yaml
 ```
 
+## DICOM Redactor
+
+Import adapter can be configured to use [DICOM Redactor Library](https://github.com/GoogleCloudPlatform/healthcare-deid/tree/master/offline_tools/redactorhttps://github.com/GoogleCloudPlatform/healthcare-deid/tree/master/offline_tools/redactor) to redact sensitive data contained in DICOM tags during C-STORE upload.
+The user can configure which tags to redact/remove in one of 3 ways:
+- redact_keep_list - a list of DICOM tags to keep untouched. Other tags are removed.
+- redact_remove_list - a list of DICOM tags to remove. Other tags are kept untouched.
+- redact_filter_profile - a predefined profile that will keep and remove particular tags.
+
+If enabled by setting one of above options, redactor also always regenerates the following UIDs:
```

Comment:
Phrasing edit, and added a bit more description about the new value:
"If enabled via one of the above options, the redactor also replaces the following UIDs with a random value:"

Reply:
Ok, but a little nitpick: resulting value is fully determined by old one, at least in current implementation of redactor (and this property is important to integration and unit tests).

Reply:
Good point, let's revert it back from:
"the redactor also replaces the following UIDs with a random value:"
to
"the redactor also always regenerates the following UIDs:"

Reply:
Ok